### PR TITLE
For #1506: Show resume in /join if user already has resume.

### PR DESCRIPTION
**Pull request by paulodamaso**
For #1506: This ticket is too big to be resolved in only 30 minutes, so I've had to break it:
- Added puzzle for creating `Resumes.resume(login)`
- Extracted `/join-post` logic into `TkJoinPost` and left `/join` logic into `TkJoin`
- Fixed `TkApp` according new `TkJoin` and `TkJoinPost` classes
- Added `resume.xsl`
- Added todo for continuing implementation.

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#1562](https://github.com/zerocracy/farm/issues/1562) is assigned to @krzyk[/z](https://www.0crat.com/u/krzyk), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/2200445113); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/main/java/com/zerocracy/pmo/Resumes.java [hunk 1]
```diff
@@ -27,6 +27,11 @@
 /**
  * Resumes.
  * @since 1.0
+ *
+ * @todo #1506:30min Implement resumes.resume(login), which will return the
+ *  resume sent for some user. It will have to return a Map mapping the resume
```
Comment (krzyk) [inline src/main/java/com/zerocracy/pmo/Resumes.java]: @paulodamaso `Map` is a poor choice, I would use `Resume` object here

**Comment by paulodamaso** [comment]
@krzyk Thanks, take a look at the fixes.

**Comment by krzyk** [comment]
@paulodamaso thanks

**Comment by krzyk** [comment]
@rultor merge

**Comment by g4s8** [comment]
@rultor merge

**Comment by krzyk** [comment]
@g4s8 please try again, looks like unrelated problem

**Comment by g4s8** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job was finished in 15 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @krzyk[/z](https://www.0crat.com/u/krzyk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#1562](https://github.com/zerocracy/farm/issues/1562) is now out of scope

**Comment by 0crat** [comment]
Job [#1562](https://github.com/zerocracy/farm/issues/1562) is pending quality review, can't start a new order

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by ypshenychka** [comment]
@0crat quality good

**Comment by 0crat** [comment]
Order was finished, quality is "good": +25 point(s) just awarded to @krzyk[/z](https://www.0crat.com/u/krzyk)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**